javascript/trees/treemax/README.md

```diff
@@ -2,10 +2,13 @@
 
 ## Whiteboard Process
 
-![whiteboard image](/javascript/linked-list/)
+![whiteboard image](/javascript/trees/treemax/cc16wb.png)
 
 ## Approach & Efficiency
 
+time: O(n)
+space: O(1)
```

Comment:
Because you are recursively traversing the tree, your call stack means that space efficiency will be O(h), not O(1).